packages/linear-event-transport/src/LinearIssueTrackerService.ts

```diff
@@ -0,0 +1,821 @@
+			throw err;
+		}
+	}
+
+	/**
+	 * Fetch a single workflow state by ID.
+	 */
+	async fetchWorkflowState(stateId: string): Promise<WorkflowState> {
+		return await this.linearClient.workflowState(stateId);
+	}
+
+	// ========================================================================
+	// USER OPERATIONS
+	// ========================================================================
+
+	/**
+	 * Fetch a user by ID.
+	 */
+	async fetchUser(userId: string): Promise<User> {
+		return await this.linearClient.user(userId);
+	}
+
+	/**
+	 * Fetch the current authenticated user.
+	 */
+	async fetchCurrentUser(): Promise<User> {
+		return await this.linearClient.viewer;
+	}
+
+	// ========================================================================
+	// AGENT SESSION OPERATIONS
+	// ========================================================================
+
+	/**
+	 * Create an agent session on an issue.
+	 * Uses native SDK method instead of raw GraphQL.
+	 */
+	async createAgentSessionOnIssue(
+		input: AgentSessionCreateOnIssueInput,
+	): Promise<AgentSessionCreateResponse> {
+		try {
+			// Use native SDK method
+			const result = await this.linearClient.agentSessionCreateOnIssue({
+				issueId: input.issueId,
+				externalLink: input.externalLink,
+			});
+
+			if (!result.success) {
+				throw new Error("Linear API returned success=false");
+			}
+
+			const agentSession = await result.agentSession;
+			if (!agentSession) {
+				throw new Error("Linear API did not return an agent session");
+			}
+
+			return {
+				success: result.success,
+				agentSessionId: agentSession.id,
+				lastSyncId: result.lastSyncId,
+			};
+		} catch (error) {
+			const err = new Error(
+				`Failed to create agent session on issue ${input.issueId}: ${error instanceof Error ? error.message : String(error)}`,
+			);
+			if (error instanceof Error) {
+				err.cause = error;
+			}
+			throw err;
+		}
+	}
+
+	/**
+	 * Create an agent session on a comment thread.
+	 * Uses native SDK method instead of raw GraphQL.
+	 */
+	async createAgentSessionOnComment(
+		input: AgentSessionCreateOnCommentInput,
+	): Promise<AgentSessionCreateResponse> {
+		try {
+			// Use native SDK method
+			const result = await this.linearClient.agentSessionCreateOnComment({
+				commentId: input.commentId,
+				externalLink: input.externalLink,
+			});
+
+			if (!result.success) {
+				throw new Error("Linear API returned success=false");
+			}
+
+			const agentSession = await result.agentSession;
+			if (!agentSession) {
+				throw new Error("Linear API did not return an agent session");
+			}
+
+			return {
+				success: result.success,
+				agentSessionId: agentSession.id,
+				lastSyncId: result.lastSyncId,
+			};
+		} catch (error) {
+			const err = new Error(
+				`Failed to create agent session on comment ${input.commentId}: ${error instanceof Error ? error.message : String(error)}`,
+			);
+			if (error instanceof Error) {
+				err.cause = error;
+			}
+			throw err;
+		}
+	}
```

Comment:
this oucld literally be one line, and the params and return type of the interface should match wathever the linear method takes and returns